x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -865,6 +869,31 @@ class PackagePolicyService {
   }
 }
 
+function validatePackagePolicyOrThrow(packagePolicy: NewPackagePolicy, pkgInfo: PackageInfo) {
```

Comment:
are there any unit tests for this logic? it would be nice to test edge cases

Reply:
The function used here `validatePackagePolicy` is unit tested